### ISSUE TITLE
atlas_v3_v1_sandia model blows up where old one doesn't

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#354]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/354)) by David Conner (Bitbucket: [dcconner](https://bitbucket.org/%7B4375dc4a-9371-4c51-973e-1f1d045590c5%7D/)).**

The original report had attachments: [atlas_v1_2013-09-17-19-24-35.bag](https://osrf-migration.github.io/drcsim-gh-pages/data/repositories/osrf/drcsim/issues/354/attachments/atlas_v1_2013-09-17-19-24-35.bag), [atlas_v3_v1_2013-09-17-19-29-48.bag](https://osrf-migration.github.io/drcsim-gh-pages/data/repositories/osrf/drcsim/issues/354/attachments/atlas_v3_v1_2013-09-17-19-29-48.bag)

----------------------------------------
When we invoke our controller to start sending commands, in this case STAND, the new 

```
#!c++
roslaunch atlas_utils atlas_v3_with_v1_inertia_sandia_hands.launch
```
"blows up", where the original doesn't.
```
#!c++

  roslaunch atlas_utils atlas_sandia_hands.launch
```
Attached are two bag files.  The atlas_v1*.bag is from sending commands to old version, and we get the expected result (nothing since it's already in stand on start).

Replaying the command outputs from that bag into the new version blows up.  See atlas_v3_v1*.bag.

Replaying into original works as expected.

**Comment by osrf-migration** [comment]
**Original comment by David Conner (Bitbucket: [dcconner](https://bitbucket.org/%7B4375dc4a-9371-4c51-973e-1f1d045590c5%7D/)).**

----------------------------------------
FYI:  To be sure it wasn't an issue with sending data too fast, we put  a sleep in controller loop to slow output down to 20 then 10 Hz.  Seems to go unstable as soon as first command is issued.

**Comment by osrf-migration** [comment]
**Original comment by David Conner (Bitbucket: [dcconner](https://bitbucket.org/%7B4375dc4a-9371-4c51-973e-1f1d045590c5%7D/)).**

----------------------------------------
Thanks to @stgray and Issue #355 I can now avoid my blow up issue by zeroing out the kp_velocities.

**Comment by osrf-migration** [comment]
**Original comment by David Conner (Bitbucket: [dcconner](https://bitbucket.org/%7B4375dc4a-9371-4c51-973e-1f1d045590c5%7D/)).**

----------------------------------------
Turns out that it worked if I set all kp_velocity to 0.0.

When I set just the ankle joints to 0.0, the robot slouches to one side. 
It appears the same as if the robot was in manipulate with bad kp_position for back joints.

For STAND mode, none of the gains should be used since k_efforts are all 0.0.

**Comment by osrf-migration** [comment]
**Original comment by Brian Gerkey (Bitbucket: [Brian Gerkey](https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/), GitHub: [gerkey](https://github.com/gerkey)).**

----------------------------------------
Sounds like this got resolved and was essentially a duplicate of #355.

**Comment by osrf-migration** [comment]
**Original comment by Brian Gerkey (Bitbucket: [Brian Gerkey](https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/), GitHub: [gerkey](https://github.com/gerkey)).**

----------------------------------------

* changed state from "new" to "duplicate"

Duplicate of #355.
